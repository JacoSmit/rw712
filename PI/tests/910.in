10
7
96 10 13 10 13 99 6
7
20 96 93 9 11 5 1
7
12 93 17 90 10 8 3
7
13 97 9 6 20 1 98
7
6 8 92 16 95 12 8
7
91 1 18 3 9 99 9
7
19 100 3 12 15 97 18
7
2 7 11 5 93 97 15
7
96 15 14 7 8 91 11
7
8 1 13 98 16 98 7
