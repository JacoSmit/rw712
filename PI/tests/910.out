1:
1 ~ 30
2 ~ 10
3 ~ 14
4 ~ 10
5 ~ 14
6 ~ 34
7 ~ 10
2:
1 ~ 24
2 ~ 32
3 ~ 32
4 ~ 8
5 ~ 8
6 ~ 8
7 ~ 0
3:
1 ~ 10
2 ~ 34
3 ~ 22
4 ~ 30
5 ~ 10
6 ~ 6
7 ~ 2
4:
1 ~ 8
2 ~ 32
3 ~ 8
4 ~ 8
5 ~ 24
6 ~ 0
7 ~ 32
5:
1 ~ 4
2 ~ 8
3 ~ 28
4 ~ 20
5 ~ 36
6 ~ 12
7 ~ 8
6:
1 ~ 24
2 ~ 0
3 ~ 24
4 ~ 0
5 ~ 8
6 ~ 40
7 ~ 8
7:
1 ~ 16
2 ~ 36
3 ~ 4
4 ~ 4
5 ~ 12
6 ~ 28
7 ~ 16
8:
1 ~ 2
2 ~ 10
3 ~ 12
4 ~ 6
5 ~ 28
6 ~ 36
7 ~ 20
9:
1 ~ 37
2 ~ 15
3 ~ 13
4 ~ 7
5 ~ 7
6 ~ 27
7 ~ 13
10:
1 ~ 8
2 ~ 0
3 ~ 8
4 ~ 32
5 ~ 24
6 ~ 32
7 ~ 8
